prime-router/metadata/schemas/HL7/color-labs-covid-19.schema

```diff
@@ -0,0 +1,22 @@
+---
+name: color-labs-covid-19
+description: Color Labs
```

Comment:
We have a pair of schemas that work for any HL7 sender, so sender-specific sub-schemas are no longer required. They set the appropriate sender_id and processing_mode_code, and patient_ethnicity has support for values from both the HL7 and OMB sets.

`hl7/hl7-ingest-covid-19-training` maps MSH-11 = "P" to "T" so senders in onboarding mode can't accidently send to STLTs/HHS. (lesson learned... `default: T` doesn't work since the field already has a value!)
`hl7/hl7-ingest-covid-19-prod` is then used for full production.

All of this can be done in settings, so no need to wait for prod deployments for testing or go live.